persistent/list.py

```diff
@@ -49,43 +52,62 @@ class PersistentList(UserList, persistent.Persistent):
     __super_clear = (
         UserList.clear
         if hasattr(UserList, 'clear')
-        else lambda inst: inst.__delitem__(slice(None, None, None))
+        else lambda inst: inst.__delitem__(_SLICE_ALL)
     )
 
+    if (2, 7, 0) < sys.version_info[:3] < (3, 7, 4):
```

Comment:
Don't think we care about 2.6.x. Simpler as:
```suggestion
    if sys.version_info[:3] < (3, 7, 4):
```


or did you mean to be leaving the native `__getitem__` in place for 2.7.x?  In which case:
```suggestion
    if (3, 0. 0) < sys.version_info[:3] < (3, 7, 4):
```

Reply:
Ah, thank you. I did mean to leave the original version in place for Python 2. This condition was left over from an earlier iteration (it was originally combined with the condition below).